Casks/m/mixing-station.rb

```diff
@@ -0,0 +1,28 @@
+cask "mixing-station" do
+  version "2.2.0"
+  sha256 :no_check
+
+  url "https://mixingstation.app/backend/api/web/download/attachment/mixing-station-pc/release/OSX"
+  name "Mixing Station"
+  desc "Audio Mixer Controller"
+  homepage "https://mixingstation.app/"
+
+  livecheck do
+    url "https://mixingstation.app/backend/api/web/changelogs/milestones"
+    strategy :json do |json|
+      json["data"]&.select { |item| item["variant"] == "Desktop" }
+                  &.map { |item| item["current"]["version"] }
```

Comment:
```suggestion
      json["data"]&.select { |item| item["variant"] == "Desktop" }
                  &.map { |item| item.dig("current", "version") }
```

Another possibility that might be easier to extend in the future:

```suggestion
      json["data"]&.map do |item|
        next if item["variant"] != "Desktop"

        item.dig("current", "version")
      end
```